mycity/mycity/mycity_controller.py

```diff
@@ -0,0 +1,202 @@
+"""
+Controller for mycity voice app.
+
+This class handles all voice requests.
+"""
+
+from __future__ import print_function
+from mycity.mycity_data_model import MyCityDataModel
+from .intents.user_address_intent import set_address_in_session, \
+    get_address_from_session, create_set_address_intent_response, \
+    request_user_address_response
+from .intents.trash_intent import get_trash_day_info
+from .intents.unhandled_intent import unhandled_intent
+from .intents.snow_parking_intent import get_snow_emergency_parking_intent
+from .intents.get_alerts_intent import get_alerts_intent
+from .intents import intent_constants
+
+
+class MyCityController:
+    """
+    Handles requests for the MyCity voice app.
+
+
+    @type mcr: MyCityDataModel
+    @param mcr: Request from platform as a MyCityRequestModel object
+    """
+    LOG_CLASS = '\n\n[class: MyCityController]'
+
+    def __init__(self, mcd):
+        """
+        Construct the controller.
+
+        @type mcd: MyCityDataModel
+        @param mcd: Request from platform as a MyCityRequestModel object
+        """
+        self._mcd = mcd
+
+
+    def main(self):
+        """
+        Route the incoming request based on type (LaunchRequest, IntentRequest,
+        etc.) The JSON body of the request is provided in the event parameter.
+
+        @type mcr: MyCityDataModel
+        @param mcr: Request from platform as a MyCityRequestModel object
```

Comment:
Since this is passed in the constructor, we don't need these here.

Reply:
The doc comments need a full overhaul. I removed this here.